WordPress/WordPressShareExtension/ShareModularViewController.swift

```diff
@@ -622,17 +673,22 @@ fileprivate extension ShareModularViewController {
 
 fileprivate extension ShareModularViewController {
     enum ModulesSection: Int {
-        case summary = 0
+        case tags = 0
```

Comment:
If we're just numbering these sequentially, you can omit the numbers here and it'll do it automatically.